webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -243,6 +245,17 @@ class ManageManifestModal extends Component {
                             >
                               <Icon type="pf" name="info" />
                             </OverlayTrigger>
+                            <div id="manifest-toggle-sca-switch">
+                              <SimpleSwitch
+                                toggleSCA={this.props.toggleSCAManifest}
+                                hideModal={this.hideModal}
+                                simpleContentAccess={simpleContentAccess}
+                                canToggleSimpleContentAccess={
```

Comment:
You'll have to check but I think `canToggleSimpleContentAccess` can be set like this: `canToggleSimpleContentAccess={disableManifestActions}` depending on how disableManifestActions is defined